emark/models.py

```diff
@@ -18,8 +18,12 @@ class Send(models.Model):
         related_name="emark_emails",
         null=True,
     )
-    from_address = models.EmailField()
-    to_address = models.EmailField()
+    # In RFC 2822 from is a mailbox-list, but Django only support a single
```

Comment:
```suggestion
    # In RFC 2822 "from" is a mailbox list, but Django only supports a single address
```

Reply:
The type is called `mailbox-list`. I will slightly alter the message.